gui/wxpython/vnet/vnet_data.py

```diff
@@ -1053,17 +1053,15 @@ def GetLastModified(self):
             "head",
         )
         try:
-            head = open(headPath)
-            for line in head:
-                i = line.find(
-                    "MAP DATE:",
-                )
-                if i == 0:
-                    head.close()
-                    return line.split(":", 1)[1].strip()
-
-            head.close()
-            return ""
+            with open(headPath) as head:
+                for line in head:
+                    i = line.find(
+                        "MAP DATE:",
```

Comment:
The comma, even if there initially, isn't really needed. By removing it, maybe ruff format can format it in the same line.